.drone.yml

```diff
@@ -5873,9 +5408,4287 @@ steps:
       from_secret: PRODUCTION_QUAYIO_DOCKER_PASSWORD
     QUAY_USERNAME:
       from_secret: PRODUCTION_QUAYIO_DOCKER_USERNAME
+    STAGING_QUAY_PASSWORD:
+      from_secret: QUAYIO_DOCKER_PASSWORD
+    STAGING_QUAY_USERNAME:
+      from_secret: QUAYIO_DOCKER_USERNAME
+  volumes:
+  - name: dockersock
+    path: /var/run
+  depends_on:
+  - Tag and push "public.ecr.aws/gravitational/teleport-ent:10-fips-amd64" to ECR
+    - production
+services:
+- name: Start Docker
+  image: docker:dind
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
+volumes:
+- name: dockersock
+  temp: {}
+
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/container_images.go:123
+################################################
+
+kind: pipeline
+type: kubernetes
+name: teleport-container-images-current-version-cron
+trigger:
+  cron:
+    include:
+    - teleport-container-images-cron
+  repo:
+    include:
+    - gravitational/teleport
+workspace:
+  path: /go
+clone:
+  disable: true
+steps:
+- name: Wait for docker
+  image: docker
+  commands:
+  - timeout 30s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
+  volumes:
+  - name: dockersock
+    path: /var/run
+- name: Build teleport image "teleport-v10-amd64"
+  image: docker
+  commands:
+  - apk --update --no-cache add curl
+  - curl -Ls -o "/go/build/Dockerfile-cron" "https://raw.githubusercontent.com/gravitational/teleport/${DRONE_SOURCE_BRANCH:-master}/build.assets/Dockerfile-cron"
+  - mkdir -p "/go/build" && cd "/go/build"
+  - docker build --target teleport --platform linux/amd64 --tag teleport-v10-amd64
```

Comment:
since we are just downloading the assets from Teleport s3, can't we just use `docker buildx`?
```
docker buildx build \
--platform linux/arm/v7,linux/arm64/v8,linux/amd64
```